src/Protocol/Diaspora.php

```diff
@@ -3204,7 +3200,7 @@ private static function sendParticipation(array $contact, array $item)
 		$author = self::myHandle($owner);
 
 		$message = ["author" => $author,
-				"guid" => System::createGUID(32),
+				"guid" => System::createUUID(),
```

Comment:
Is this valid with Diaspora's expected data size for a `guid`?

Reply:
Oh yeah. Just have a look at the Godzilla-size GUIDs from Hubzilla :-)

I had a discussion about the difference between GUID and UUID (which SocialHome is using) and thought that a UUID-like identifier would look nice as well - and could possibly later be transmitted via AP in the UUID field.

Reply:
I'm fine with the change as long as it doesn't break compatibility with the remote systems that aren't creating UUIDs at the moment.